src/test/kotlin/org/rust/lang/core/completion/RsCompletionTest.kt

```diff
@@ -150,6 +150,14 @@ class RsCompletionTest : RsCompletionTestBase() {
         use self::foo::quux/*caret*/;
     """)
 
```

Comment:
Let's add another test for glob imports:

    use self::{f}


Here, the semicolon should not be added, but nothing should explode as well

Reply:
So the test would be 
```kotlin
fun `test use glob no semicolon`() = doSingleCompletion("""
        mod foo { pub fn quux() {} }
        use self::{f/*caret*/}
    """, """
        mod foo { pub fn quux() {} }
        use self::{f/*caret*/}
    """)
```
?

Reply:
Rater

```

    fun `test some illuminating name`() = doSingleCompletion("""
        use self::{f/*caret*/}
        fn foo() {}
    """, """
        use self::{foo/*caret*/}
        fn foo() {}
    """)
```

Reply:
I couldn't use your suggestion because it is not a single completion. I just put the function in a module.